longevity_operator_multi_tenant_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2022 ScyllaDB
+
+import copy
+
+from longevity_test import LongevityTest
+
+from sdcm.utils.common import ParallelObject
+
+
+# pylint: disable=too-many-instance-attributes
+class ScyllaClusterStats(LongevityTest):
+    # pylint: disable=too-many-arguments,super-init-not-called
+    def __init__(self, db_cluster, loaders, monitors, prometheus_db, params, test_config, cluster_index):
+        self.db_cluster = db_cluster
+        self.loaders = loaders
+        self.monitors = monitors
+        self.prometheus_db = prometheus_db
+        self.params = copy.deepcopy(params)
+        self._es_doc_type = "test_stats"
+        self._stats = self._init_stats()
+        self.test_config = test_config
+        self._duration = self.params.get(key='test_duration')
+        self.create_stats = self.params.get(key='store_perf_results')
+        self.status = "RUNNING"
+        self.cluster_index = str(cluster_index)
+        self._test_id = self.test_config.test_id() + f"--{cluster_index}"
+        self._test_index = self.get_str_index()
+        self.create_test_stats()
+        self.start_time = self.get_test_start_time()
+
+    def get_str_index(self):
+        return f"k8s-longevity-{self.db_cluster.k8s_cluster.tenants_number}-tenants"
+
+    def id(self):  # pylint: disable=invalid-name
+        return self.test_config.test_id() + f"-{self._test_index}"
+
+    def __str__(self) -> str:
+        return self._test_index + f"--{self.cluster_index}"
+
+    def __repr__(self) -> str:
+        return self.__str__()
+
+
+class LongevityOperatorMulitiTenantTest(LongevityTest):
+    scylla_clusters_stats = []
+    test_type = "longevity"
```

Comment:
what `test_type` is used for ? (i.e. I don't see any usage of it)

Reply:
removed. It's relevant for statistics and not relevant for longevity